index.html

```diff
@@ -0,0 +1,396 @@
+                <path d="M20 2.20496V3.221C20 3.74239 19.6792 4.19694 19.1979 4.37742L4.33158 9.87207H4.32489L0.0468192 11.4429C0.0267657 11.4496 0.0133984 11.4429 0.00671387 11.4229V8.67555C0.00671387 8.41485 0.0936106 8.16753 0.247354 7.96031C0.274092 7.9202 0.300831 7.88009 0.340938 7.84667C0.374361 7.79319 0.427834 7.75309 0.487994 7.7063C0.581577 7.62608 0.695215 7.56592 0.815536 7.52581L1.25003 7.37207V7.36539L1.28345 7.3587L18.3356 1.06191C19.1444 0.754423 20 1.34935 20 2.20496Z" fill="#0ACF83"/>
+                <path d="M17.6872 7.97369L7.54679 11.7237L7.51337 11.7437H7.50668L1.83155 13.836C1.65776 13.8961 1.50401 13.9764 1.35695 14.0633C0.528074 14.5579 0.00668694 15.467 0.00668694 16.463V17.4523L0.975938 17.0913L1.75134 16.8039L7.16578 14.8052L10.5815 13.5419L15.4412 11.7504L17.5735 10.975C17.8275 10.8814 18.0749 11.0686 18.0749 11.3293V12.9937C18.0749 13.1475 17.988 13.2945 17.8342 13.348L13.1083 15.086L11.2567 15.7678L2.0254 19.1702L0.969251 19.5579L0 19.9122V30.0258C0 30.086 -8.16775e-07 30.1461 0.0200527 30.2063C0.0267371 30.2732 0.0401061 30.3534 0.0534751 30.4202C0.160427 30.9149 0.46123 31.336 0.902406 31.5967L1.06952 31.6903L1.06284 31.6702C0.648397 31.3694 0.381017 30.8748 0.381017 30.3133C0.381017 29.7852 0.621658 29.3173 1.00267 29.0165C1.10963 28.9296 1.21658 28.8627 1.3369 28.8026C1.44385 28.7491 14.3249 24.0098 14.3249 24.0098L18.4692 22.479L18.877 22.332C19.5521 22.078 19.9933 21.4429 19.9933 20.7344V7.12476L17.6872 7.97369ZM5.06016 24.3707C5.06016 24.598 4.9131 24.7919 4.70588 24.8721L3.52273 25.3066C3.17513 25.4336 2.80749 25.1796 2.80749 24.8052V23.8026C2.80749 23.5753 2.94786 23.3814 3.15508 23.3012L4.32487 22.8667C4.68583 22.7397 5.06016 22.9937 5.06016 23.3681V24.3707Z" fill="#0ACF83"/>
+            </svg>
+            <h1>E-Cards</h1>
+        </div>
+        <nav>
+            <ul>
+                <li>
+                    <a href="#">Transactions</a>
+                </li>
+                <li>
+                    <a href="#">Cards</a>
+                </li>
+            </ul>
+        </nav>
+        <div>
+            <img src="../img/profile_logo.png" alt="">
+            <div>
+                <span>Itai Bracha</span>
+                <a href="mailto:Itaibrahca31@gmail.com">Itaibrahca31@gmail.com</a>
+            </div>
+        </div>
+        <nav>

```

Comment:
Just a remark. You can put nav links as li inside ul, but as a rule `<a>`-s  just put inside nav and that's just fine)